docs/WritingTests.asciidoc

```diff
@@ -1277,8 +1277,25 @@ test module.
 
 ==== Enable snapshots for each module
 
-* Run the worker with --no-cleanup parameter. This will preserve the hard
- disks after test runs.
+* Run the worker with `--no-cleanup` parameter. This will preserve the hard
+ disks after test runs. If you are starting the worker(s) via the systemd unit,
```

Comment:
I know that this is not part of your change but nevertheless: Using `--no-cleanup` can generally cause problems. That hard disks are preserved is certainly not the only effect one should be aware of (e.g. jobs from a previous run might be wrongly uploaded for the next run as well, old status files are picked up leading to unintended behavior).

Reply:
Oh! I did not know that. That would actually explain all the weird quirks that I've been observing on my local openQA instance after I've added `--no-cleanup` to all the workers. I'll do some further testing if the issues disappear when I remove them.

But that means that I can have either snapshots and quirks or no snapshots? There's no setting like `--no-cleanup-harddrives`?

Reply:
I think so, there's no such a setting.